26-register_registration/creator_navigates_to_resource_type_tab.feature

```diff
@@ -2,7 +2,7 @@ Feature: Creator navigates to Resource Type tab
 
   @274
   Scenario: Creator navigates to the Resource Type tab and selects Resource type "Contribution to journal"
-    Given Creator begins registering a Registration in the Wizard
+    Given Creator begins registering a Registration in the Wizard with a file
```

Comment:
trengs egentlig denne utvidelsen? alle scenarioene her gjelder vel både for start med fil og link

Reply:
Nei, starter du registrering med fil så dukker ikke DOI-feltet opp

Reply:
Et annet problem for testing er at starter man med lenke så er Resource type og subtype allerede fylt ut og minst en av testene sjekker subtype for required

Reply:
Ja, forsåvidt...
bare vær klar over at DOI-feltet vil dukke opp igjen om Creater spør om DOI etterpå, og at en DOI ikke nødvendivis har type